docs/integration_resources/software/nuke.md

```diff
@@ -14,59 +14,107 @@ nshift.show()
+| *Float*      | -              | Knob                                             | Double_Knob                                                  |                                                                                                     |
+| *Instance*   | image*         | Input/Output<br/>Knob<br/>Knob                   | - <br/>Channel_Knob <br/>File_Knob                           | Provides the input image like a single image. (Current Frame)                                       |
+| *Instance*   | images*        | Output<br/>Knob<br/>Knob                         | - <br/>File_Knob                                             | Provides the input image like a single image. (Current Frame)                                       |
+| *Instance*   | imageSequence* | Input/Output<br/>Knob<br/>Knob<br/>Knob<br/>Knob | - <br/>Channel_Knob <br/>File_Knob<br/>Int_Knob<br/>Int_Knob | Provides the input image like a image sequence rendering the full frame range provided in the knob. |
+| *Instance*   | -              | Knob                                             | EvalString_Knob                                              | Require custom implementation                                                                       |
+| *Int*        | -              | Knob                                             | Int_Knob                                                     |                                                                                                     |
+| *List*       | points*        | Input                                            | -                                                            | Requires to connect a Tracker node to get the points from.                                          |
+| *List*       | -              | Knob                                             | EvalString_Knob                                              | Require custom implementation                                                                       |
+| *Object*     | -              | Knob                                             | EvalString_Knob                                              | Require custom implementation                                                                       |
+| *String*     | -              | Knob                                             | EvalString_Knob                                              |                                                                                                     |
+
+### Node config
+
+The node config have 3 options:
+- autoClean: Removes automatically all the temporal renders done for the input when the executiong ends.
+- backupOutputImage: Copy the output image in 
+- imageType: Can be PIL or np.array. This knob specifies the image instance type to use to provide and read from the input and output from the workflow. If the workflow expects or return a difference type it may not work.

```

Comment:
```suggestion
The node configuration has three options:
- autoClean: Automatically removes all temporary renders created for the input once the execution ends.
- backupOutputImage:  Copies the output image.
- imageType: Specifies the format used to write and read the inputs and outputs for the workflow.  The instance types can be either PIL or np.array.  If the workflow expects or returns a different type, it may not work correctly.
```